main.py

```
@@ -1134,0 +1,1230 @@
+            url: The URL to scrape
+            skip_cache: Whether to skip the cache and force a new request
+
+        Returns:
+            The HTML content as a string
+
+        Raises:
+            requests.exceptions.RequestException: If the request fails after retries
+        """
+        import time
+        import tracemalloc
+        import psutil  # type: ignore
```

Ruff (0.8.2)

`psutil` imported but unused

Remove unused import: `psutil`

(F401)